packages/cli/src/commands/integration/add.ts

```diff
@@ -380,13 +392,95 @@ async function confirmProductSelection(
   return client.input.confirm('Confirm selection?', true);
 }
 
+async function getAuthorizationId(
+  client: Client,
+  teamId: string,
+  installation: IntegrationInstallation,
+  product: IntegrationProduct,
+  metadata: Metadata,
+  billingPlan: BillingPlan
+): Promise<string> {
+  output.spinner('Validating payment...', 250);
+  const originalAuthorizationState = await createAuthorization(
+    client,
+    installation.integrationId,
+    installation.id,
+    product.id,
+    billingPlan.id,
+    metadata
+  );
+
+  if (!originalAuthorizationState.authorization) {
+    output.stopSpinner();
+    output.error(
+      'Error during authorization, failed to get an authorization state.'
+    );
+    throw new Error('Failed to get an authorization state');
```

Comment:
What action should a customer take if this see this error? Or do we catch elsewhere and error something else?

Reply:
This particular situation means our API is broken/down/there was a critical failure. Realistically, I'm not even sure if redirecting them to the dashboard would help, because that could also be down.

Reply:
We might ask them to contact support in cases like this?

Reply:
But not blocking!

Reply:
Added 🙂